### DISCUSSION TITLE
A set of global "attributes" that can live in any tag

### DISCUSSION THREAD

**Discussion by daveajones** [Podcastindex-org/podcast-namespace#602] [Ideas]
In HTML there is the notion of "global attributes" that can live in any tag:  https://html.spec.whatwg.org/#global-attributes

These are things like `style=` and `title=`.

With the recent discussion of being able to label various parts of the namespace as being generated by Artificial Intelligence, I'd like to propose that we develop a set of global attributes for the podcast namespace.  For instance, if there were a global attribute called something like `source=`, then a standard could be developed around what that attribute would mean when certain values appear in it.

```xml
<podcast:transcript 
    url="https://mp3s.nashownotes.com/PC20-165-Captions.srt"
    type="application/srt"
    source="ai"
/>
```

Don't get hung up on the specific names of things here.  This is just spit-balling to start the discussion on whether this is a good approach.  This seems preferable over declaring this sort of attribute in a dozen different places.

**Comment by agates** [comment]
I think we'd end up having to document the context of each usage per-tag so I don't see why we'd define them globally in the first place.

There may also be cases where we'd want to override the global attributes and I think it would only lead to more confusion.
